packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/AuthenticationChange.kt

```diff
@@ -0,0 +1,34 @@
+package io.realm.kotlin.mongodb
+
+/**
+ * TODO
+ */
+public sealed interface AuthenticationChange {
+    /**
+     * TODO
+     */
+    public val user: User
+    /**
+     * TODO
+     */
+    public fun didLogIn(): Boolean
```

Comment:
How come that we like to expose the state as a boolean when we already have it as an specialized interface in: `LoggedIn` or `LoggedOut`?